src/app.py

```diff
@@ -511,9 +509,18 @@ def last_activity_to_datetime(use_example_data: bool, x: str | None) -> str | No
     # Convert date column from str to datetime
     df_historic_data["day"] = df_historic_data["day"].apply(lambda x: datetime.strptime(x, "%Y-%m-%d"))
 
+    # Create a date picker for the user to select the date range
```

Comment:
It might be a good idea to display the date pickers side by side using st.columns()